engine/lib/import_thir.ml

```diff
@@ -1681,11 +1710,65 @@ and c_item_unwrapped ~ident ~drop_body (item : Thir.item) : item list =
             ~f:(fun { attributes; _ } -> not (should_skip attributes))
             items
         in
-        let has_type =
-          List.exists items ~f:(fun item ->
-              match item.kind with Type _ -> true | _ -> false)
+        let items =
+          if erased then
+            [
```

Comment:
Can you rewrite this from `[ the_item_expr ]` to `let dummy_assoc_item = the_item_expr in [dummy_assoc_item]`?
Or add a comment.
But we should convey the idea that here we're producing a dummy associated item on purpose